### PR TITLE
 home page&about page

### PR DESCRIPTION
**Pull request by aakbarkhan**
# Description:

  * Features Mobile: up to 768px wide.
  * Added mobile CSS styles
  * Added desktop CSS styles
  * Created dynamic HTML page in JavaScript.
  * Created dynamic Mobile Menu in JavaScrip.
  * Google chrome is used for the project.

[Live Link for the page](https://aakbarkhan.github.io/capstone-project/)

[Loom Video Link](https://www.loom.com/share/a9ca01076c7a4c8099b999f66eb23b61)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,64 +11,128 @@
...
-            <p>Listen to the speakers for various countries about the messages  of sharing and opening.</p>
-
-
-        </article>
-
+        <p class="all-list">SEE THE WHOLE PROGRAME</p>
+        <button type="button" class="join">
+            Join CG Tech Summit 2021
+        </button >
     </section>
```
Comment (rloterh) [inline index.html]: The **typography** and **spacing** of your main program section do not follow the [template design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). Kindly ensure to improve upon the following:
- [x] The **font family** (san-serif type) of the _heading text_ in red and the white _description text_ should match the font provided in the template design.
- [x] There should be enough or **appropriate left and right padding**/spacing around the contents of each program, as can be seen in the template design.

**Your main program section:**
image



**The original main program section:**
image

File: menu.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+const menu = document.querySelector('.nav');
+const menuItems = document.querySelectorAll('.nav-link');
+const hamburger = document.querySelector('.hamburger');
+const closeIcon = document.querySelector('.closeIcon');
+const menuIcon = document.querySelector('.menuIcon');
+closeIcon.style.display = 'none';
+
+function toggleMenu() {
```
Comment (rloterh) [inline menu.js:8]: - [x] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.

File: index.html [hunk 3]
```diff
@@ -11,64 +11,128 @@
...
+    <section class="presenters">
+        <div class="speaker-heading">
+            <h3 class="speaker-title"> Featured Speakers</h3>
+            <hr class="line">
+        </div>
+        <div id="presenters-card">
+        </div>
+    </section>
```
Comment (rloterh) [inline index.html]: The typography of your _Featured Speakers_ sections does not match the provided design. Kindly note that we actually want this design to as much as possible mimic the provided  [design template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). Kindly ensure to improve upon the following:
- [x] The font-family types should be of a san-serif, as can be seen in the template design
- [x] The _text description_ below the names of the main speakers in reddish color should be **italicized**, as can be seen in the template design:

**Your Featured speakers section:**
image

**The original Featured speakers section:**
image

File: about.html [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+    </nav>
+    <nav class="nav-links">
+        <div >
+            <a  class="logo" href="index.html">
+                <img src="./assets/logoSummit.png" alt="">
+            </a>
+        </div>
+        <button type="button" class ="hamburger"> 
+            <i class="menuIcon fas fa-bars"> </i>
+            <i class="closeIcon fas fa-times"></i>
```
Comment (rloterh) [inline about.html:37]: - [x] The _menu items_ on your mobile menu appear **inactive** when clicked or tapped. When a mobile _menu item_ is clicked or tapped it **does not jump to the rightful section** of the page or go to the appropriate page. Kindly ensure to improve upon this aspect

File: about.html [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+        Please contact us per Email fo rany further questions about Creative Global Summit 2021! <br>
+        <b>cgtsummit2021@cgtindia.org</b> > 
+    </p>
+    <div class="prize">
+        <div class="logo-prize">
+            <h4 class="logo-info">Creative Technology Global Summit 2021 Logo</h4>
+            <hr class="line last">
+            <p class="logo-disc">The logo of creative Technology Global Summit 2021 was decide through the logo competiotion from 8, june to 7, July.<span class="about-disc">  <br> After online public voting and Selction Committee's judging process, the log from Naresh Agarwal won the prize. After consultation with Mr. Agarwal we optimize the proxe-winning summit as following:</span> </p> 
+            <div class="log-border">
+                <img src="./assets/logoSummit.png" alt="">
```
Comment (rloterh) [inline about.html:72]: - [x] OPTIONAL
It's a good practice to provide `alt` names for your images so as to improve the accessibility features of your site.  Kindly endeavor to add `alt` names to **all** your images.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Aku, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍 
- Descriptive Pull Request 
- Good presentation of the project
- Nice to recognize Cindy Shin, the author of the original design in your acknowledgment section
- Project hosted on Github pages and working
- The dynamic aspect is well implemented
- Great UI design
- Good ReadMe


## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comments marked **[OPTIONAL]** under the review._
<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 4 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (DeliceLydia) [reply index.html]: Addressed this!

File: index.html (hunk 3 above)
Comment (DeliceLydia) [reply index.html]: Resolved this well!

File: about.html (hunk 4 above)
Comment (DeliceLydia) [reply about.html:37]: Nailed this

File: about.html [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+    </ul>
+</div>
+<footer>
+    <img class="footer-img" src="./assets/logoSummit.png" alt="">
+    <p class="footer-note">2021 Creative Global Technology India. Some Rights Reserved.</p>
+</footer>
+<!-- <footer class="about-footer">
+    <img class="footer-img" src="./logoSummit.png" alt="">
+    <p class="footer-note">2021 Creative Global Technology India. Some Rights Reserved.</p>
+</footer> -->
```
Comment (DeliceLydia) [inline about.html]: - [x] Remove commented codes
- [x] Fix linters before submitting

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @aakbarkhan, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
- [ ] Consider writing a professional PR which is well structured like giving it 
### Description
-
-

### Features
-
-
etc..


### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rukundoeric — APPROVED** [review]
Hi @aakbarkhan,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**